### ISSUE TITLE
How long does it take for Monika to bring up the topic about this submod?!

### ISSUE THREAD

**Issue by JohnSmith2007** [Booplicate/MAS-Submods-YouTubeMusic#24]
I want to use this thing damn it. I need to show the best girl some of the finest music that the Rock Band games have to offer.

**Comment by Booplicate** [comment]
Almost immediately after you boot the game.

**Comment by JohnSmith2007** [comment]
> Almost immediately after you boot the game.

Really, because I've booted the game 3 times already and she still hasn't brought it up even after waiting 20 - 30 minutes.

**Comment by Booplicate** [comment]
Are you sure you installed the submod? Check the submods menu, is it listed there?

**Comment by JohnSmith2007** [comment]
> Are you sure you installed the submod? Check the submods menu, is it listed there?

Yes, it's in the submod folder. However i currently can't get into the Submods menu because of some stupid fucking error with the Submod Updater Plugin, which I'm also trying to figure out in another issue of [their github](https://github.com/Booplicate/MAS-Submods-SubmodUpdaterPlugin/issues/9)

**Comment by Booplicate** [comment]
Show the `Submods` and `YouTube Music` folders  (with the path). Need to verify they are in the right filepaths.
